Datadog.Trace.sln

```diff
@@ -316,7 +316,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Datadog.Trace.ClrProfiler.M
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DogStatsD.RaceCondition", "reproductions\DogStatsD.RaceCondition\DogStatsD.RaceCondition.csproj", "{AAD9038B-8F16-4E09-93AF-B13E9F7ACD66}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DomainNeutralAssemblies.App.NoBindingRedirects", "reproduction-dependencies\DomainNeutralAssemblies.App.NoBindingRedirects\DomainNeutralAssemblies.App.NoBindingRedirects.csproj", "{4BB17962-8146-462F-8657-0DD4C2FFFFB7}"
```

Comment:
Looks like project `DomainNeutralAssemblies.App.NoBindingRedirects` was removed from the solution, but the files were not deleted. Was this intentional?

Reply:
That was a merge error. I'll fix it.